pages/index.tsx

```diff
@@ -56,8 +67,24 @@ export const HomePage: NextPage = () => {
       )
     }
 
+    if (selectedNeighborhoods.length > 0) {
+      const neighborhoodResources: string[] = []
+      selectedNeighborhoods.forEach(function (neighborhood: Neighborhood) {
+        if (neighborhood['Resources']) {
+          neighborhood['Resources'].forEach(function (resourceId: string) {
+            neighborhoodResources.push(resourceId)
+          })
+        }
+      })
+      filteredResources = filteredResources.filter(function (
+        resource: Resource
+      ) {
+        return neighborhoodResources.includes(resource['id'])
+      })
```

Comment:
Hmm, getting all resources associated with the the selected neighborhoods and then filtering resources based on those resources seems a bit roundabout? Maybe we could add a filter the resources based on a neighborhoods field (which exists in the resources Airtable but not on the `Resource` type because I just never added all the fields to that type that were in the Airtable because there are a lot and we don't use all of them). 

The field on the `Resource` type could look like:

```
export type Resource = {
    id: string
    ...
    neighborhoodNames?: string[]
    ...
}
```

Reply:
refactored it to use this method, good call out